opm/output/eclipse/AggregateWellData.hpp

```diff
@@ -39,6 +40,12 @@ namespace Opm { namespace data {
 }} // Opm::data
 
 namespace Opm { namespace RestartIO { namespace Helpers {
+    
+    struct ActionResStatus {
```

Comment:
This is just a small drive-by comment - not important either; but anyway. In situations where you need to create a list of several pieces of information you have (at least) two alternatives:

1. You can create several lists of scalar.
2. You can create one list of a composite object.

In the `ActionResStatus` case you have chosen to create *two* lists of `Action::Result` and `std::string`objects respectively. The disadvantage with this approach is that the modelling/implementation does not focus on the smallest interesting item - namely a pair of `std::string` and `Action::Result`- the item actually does not even exist in the implementation. In my opinion it is generally[1] a good rule to flip things around, and make one of list of objects, in this case its could be as simple as:
```
std::vector<std::pair<std::string, Action::Result>>
```
or maybe:
```
Opm::OrderedMap<std::string, Action::Result>
```
or the more general:
```
struct ActionStatus {
      Opm::Action::Result result;
      std::string name;
}

std::vector<ActionResult> result;
```

[1]: Of course - this is no hard and fast rule, rather an opinion from my side. Obvious exceptions are when one of the vectors - e.g. the list of names in this case - should be used in isolation.